PluralKit.Bot/Commands/MemberAvatar.cs

```diff
@@ -87,13 +119,26 @@ public async Task Avatar(Context ctx, PKMember target)
         await AvatarCommandTree(AvatarLocation.Member, ctx, target, guildData);
     }
 
+    public async Task WebhookAvatar(Context ctx, PKMember target)
+    {
+        var guildData = ctx.Guild != null
+            ? await ctx.Repository.GetMemberGuild(ctx.Guild.Id, target.Id)
+            : null;
+
+        await AvatarCommandTree(AvatarLocation.MemberWebhook, ctx, target, guildData);
+    }
+
     private async Task AvatarCommandTree(AvatarLocation location, Context ctx, PKMember target,
                                          MemberGuildSettings? guildData)
     {
         // First, see if we need to *clear*
-        if (ctx.MatchClear() && await ctx.ConfirmClear(location == AvatarLocation.Server
-                ? "this member's server avatar"
-                : "this member's avatar"))
+        if (ctx.MatchClear() && await ctx.ConfirmClear("this member's " + location switch
+        {
```

Comment:
there's 3 of these `location switch` now, can you put them in a function somewhere or maybe set the values directly in the enum?